Source/stores.cpp

```diff
@@ -2608,10 +2606,10 @@ void StorePrior()
 	PlaySFX(IS_TITLEMOV);
 	if (stextsel != -1 && stextscrl) {
 		if (stextsel == stextup) {
-			if (stextsval != 0)
+			if (stextsval != 0) {
 				stextsval -= 4;
-			if (stextsval < 0)
-				stextsval = 0;
+			}
+			stextsval = std::max(stextsval, 0);

```

Comment:
```suggestion
			stextsval = std::max(stextsval - 4, 0);
```
The if != 0 statement is unnecessary, the decrement can be inlined and let std::max clamp to 0 in all cases.

Webkit Code Style guidelines forbid braces on single statement conditionals as well. (I prefer braces everywhere but try keep it consistent for this project 😅 )

Reply:
Updated the pull request and looking into Webkit Code Style guideline (https://webkit.org/code-style-guidelines/). Thank you @ephphatha !

Reply:
> I prefer braces everywhere

Same :D

Reply:
@bcko if you haven't see it we have some additional rules on top of what Webkit provides: https://github.com/diasurgical/devilutionX/wiki/Code-Style